src/kvstore/kvstore_dist.h

```diff
@@ -367,7 +396,7 @@ class KVStoreDist : public KVStoreLocal {
 #if MKL_EXPERIMENTAL == 1
       mkl_set_tblob_eager_mode(send_buf.data());
 #endif
-      real_t* data = send_buf.data().dptr<real_t>();
+      DType* data = send_buf.data().dptr<real_t>();
```

Comment:
Why send_buf get dptr data with real_t type when it get dptr data with DType type in line 327?
I got a compiler's error here.